nmn.Templates/src/interpolation.js

```diff
@@ -50,18 +49,44 @@ async function processTag(
   return await getInput(`Value for ${tag}`)
 }
 
+// Apply any matching tag functions
+export async function processTags(
```

Comment:
This function is defined twice

Reply:
Exists in `interpolation.js` and `tagFunctions.js`

Reply:
Thanks. Hopefully the next commit sorts this.